cmd/bosun/expr/funcs.go

```diff
@@ -100,6 +100,17 @@ func tagRename(args []parse.Node) (parse.Tags, error) {
 	return tags, nil
 }
 
+func tagMetaTag(args []parse.Node) (parse.Tags, error) {
+	tags, err := tagFirst(args)
+	if err != nil {
+		return nil, err
+	}
+	newTagList := csv(args[2].(*parse.StringNode).Text)
```

Comment:
This is both ugly and unsafe. I really can't understand what it is doing, but I trust that it does. Maybe a simple comment to the effect of "third arg contains the new tag name, possibly comma separated with metadata field"